CompuCell3D/core/Demos/CompuCellPythonTutorial/EnergyReport/README.txt

```diff
@@ -0,0 +1,9 @@
+This demo is incorrect and pending fixes. 
```

Comment:
The histograms should not show negative MCS. What should the axes show instead?
![energyReport](https://github.com/CompuCell3D/CompuCell3D/assets/79017811/3ac715cc-ae6e-435b-b490-30133cc963a8)